### PR TITLE
add option to pass polling interval in milliseconds

### PR DESCRIPTION
**Pull request by santanajames**
this is in response to #126

### CODE REVIEW

**Comment by michael-ciniawsky** [comment]
```
postcss src/*.css -d dest -w --poll 200
```

```js
{ 
   usePolling: argv.poll ? true : false,
   interval: argv.poll && typeof argv.poll === 'string' ? argv.poll : 100
   // Not sure about the type either {String} or {Number} depending on how yargs parses it
}
```

**Comment by santanajames** [comment]
@michael-ciniawsky your changes were great. Just had to change the poll type from boolean to number and we're set.

File: index.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ Usage:
   })
   .option('poll', {
     desc: 'Use polling for file watching',
```
Comment (RyanZim) [inline index.js]: Description should be updated to reflect the possibility to pass a number

File: index.js [hunk 2]
```diff
@@ -196,7 +196,10 @@ Promise.resolve()
     if (argv.watch) {
       const watcher = chokidar.watch(
         input.concat(dependencies(results)),
-        { usePolling: argv.poll }
+        {
+          usePolling: argv.poll,
```
Comment (michael-ciniawsky) [inline index.js]: `argv.poll || false` ?

But if I remember right yargs defaults to `false` anyways, should be ok then

Comment (RyanZim) [inline index.js]: Yargs defaults to `undefined` IIRC, but it doesn't matter here.

Comment (santanajames) [inline index.js]: I'm fine with updating but that was my thought also.

**Comment by RyanZim** [comment]
Needed some changes; landed in https://github.com/postcss/postcss-cli/commit/78cf6001a058afa65ca32c29c1831da00698bac0.

**Pull request closed without merging**